applications/FluidDynamicsApplication/tests/cpp_tests/test_two_fluid_navier_stokes_element_volume_error_source.cpp

```diff
@@ -0,0 +1,1836 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Daniel Diez, Ruben Zorrilla
+//
+//
+
+// System includes
+#include <set>
+
+// External includes
+
+// Project includes
+#include "testing/testing.h"
+#include "containers/model.h"
+#include "spaces/ublas_space.h"
+#include "includes/properties.h"
+#include "includes/model_part.h"
+#include "utilities/math_utils.h"
+#include "includes/global_pointer_variables.h"
+#include "custom_elements/two_fluid_navier_stokes.h"
+#include "custom_constitutive/newtonian_2d_law.h"
+#include "custom_constitutive/newtonian_3d_law.h"
+#include "custom_constitutive/newtonian_two_fluid_3d_law.h"
+#include "fluid_dynamics_application_variables.h"
+#include "processes/find_nodal_neighbours_process.h"
+#include "utilities/normal_calculation_utils.h"
+
+namespace Kratos {
+	namespace Testing {
+
+		typedef ModelPart::IndexType									 IndexType;
+		typedef ModelPart::NodeIterator					          NodeIteratorType;

```

Comment:
We decided to avoid indentation in namespaces as it is more comfortable when browsing through the code. Besides, if used, our indentation standard is 4 spaces.